src/Main.java

```diff
@@ -2,5 +2,104 @@ public class Main {
     public static void main(String[] args) {
         System.out.println("Hello, World!");
 
+        System.out.println("Задача№1");
+        int a = 4000;
+        byte b = 22;
+        float c = 3.75f;
+        short d = 25645;
+        long i = 9233567899L;
+        double f = -1.7E+308;
+        System.out.println("Значение переменной a с типом Целочисленныеравно " + a);
+        System.out.println("Значение переменной b с типом Целочисленныеравно " + b);
+        System.out.println("Значение переменной c с типом Целочисленныеравно " + c);
+        System.out.println("Значение переменной d с типом Целочисленныеравно " + d);
+        System.out.println("Значение переменной i с типом Целочисленныеравно " + i);
+        System.out.println("Значение переменной f с типом Целочисленныеравно " + f);
+        System.out.println("Задача№2");
+        float m = 27.12f;
+        long v = 987678965549L;
+        double x = 2.786;
+        short k = 569;
+        int t = - 259;
+        int p = 27897;
+        byte u = 67;
+        System.out.println(m);
+        System.out.println(v);
+        System.out.println(x);
+        System.out.println(k);
+        System.out.println(t);
+        System.out.println(p);
+        System.out.println(u);
+        System.out.println("Задача№3");
+        int classLP = 23;
+        int classAS = 27;
+        int classEA = 30;
+        int numberStudents = (int) (classLP + classAS + classEA);
```

Comment:
зачем преобразование к int? у тебя и так все переменные типа int, никакого приведения делать не нужно, оно не имеет смысла никакого
```suggestion
        int numberStudents = classLP + classAS + classEA;
```